test-requirements.txt

```diff
@@ -8,5 +8,6 @@ paramiko
 pytest
 pyyaml
 wget
+xmltodict
```

Comment:
can we avoid that estra dep? and use for now just an inline string? Like we do for the other libvirt code? and if we decide to change, let's do it all together imo

Reply:
I'm no a big fan of inline string especially for long ones.
A dic also allow easier access to items if needed.
Do you think that extra dep is an issue ?

Reply:
I think it's unneeded complexity (extra dep plus accessing a path in the filesystem) given that this is only 5 lines of xml. Plus we use already inline strings.

Reply:
Done